### PR TITLE
Ensure each k/k presubmit job has a testgrid entry

### PR DESCRIPTION
**Pull request by krzyzacy**
Not doing for all presubmits because I especially don't care about the k-security jobs.

also s/jenkinsjobs/jobs/

### CODE REVIEW

File: testgrid/jenkins_verify/jenkins_validate.go [hunk 1]
```diff
@@ -75,39 +75,48 @@ func main() {
 		os.Exit(1)
 	}
 
-	// Also check prow postsubmit and periodic jobs
+	// Also check k/k presubmit, prow postsubmit and periodic jobs
+	for _, job := range prowConfig.RepoPresubmits("kubernetes/kubernetes") {
```
Comment (kargakis) [inline testgrid/jenkins_verify/jenkins_validate.go]: Expose this as a flag?

Comment (kargakis) [reply testgrid/jenkins_verify/jenkins_validate.go]: I see you have other stuff hardcoded as well. Doesn't matter for now.

**Comment by krzyzacy** [comment]
rebased and cleaned up
/cc @kargakis

**Comment by kargakis** [comment]
prow changes lgtm

**Pull request merged**